README.md

```diff
@@ -14,10 +18,57 @@ connections and responses from end endpoint.
 
 (TODO: some words to go along with these diagrams)
 
-#### Development setup
+### Development setup
+
+#### Node.js
+
+If you want to install Node.js natively visit [Node.js](http://nodejs.org), please make sure it is version 4.4.4
```

Comment:
`,` needed after `natively`. Also `please make sure it is version 4.4.4` should be its own sentence and should have a `.` at the end.